providers/aws/services/dynamodb/dynamodb_service_test.py

```diff
@@ -0,0 +1,178 @@
+        # Create DynamoDB Table
+        table = dynamodb_client.create_table(
+            TableName="test1",
+            AttributeDefinitions=[
+                {"AttributeName": "client", "AttributeType": "S"},
+                {"AttributeName": "app", "AttributeType": "S"},
+            ],
+            KeySchema=[
+                {"AttributeName": "client", "KeyType": "HASH"},
+                {"AttributeName": "app", "KeyType": "RANGE"},
+            ],
+            BillingMode="PAY_PER_REQUEST",
+        )["TableDescription"]
+        # DynamoDB client for this test class
+        audit_info = self.set_mocked_audit_info()
+        dynamo = DynamoDB(audit_info)
+        assert len(dynamo.tables) == 1
+        assert dynamo.tables[0].arn == table["TableArn"]
+
+    # Test DynamoDB Describe Table
+    @mock_dynamodb
+    def test__describe_continuous_backups__(self):
+        # Generate DynamoDB Client
+        dynamodb_client = client("dynamodb", region_name=AWS_REGION)
+        # Create DynamoDB Table
+        table = dynamodb_client.create_table(
+            TableName="test1",
+            AttributeDefinitions=[
+                {"AttributeName": "client", "AttributeType": "S"},
+                {"AttributeName": "app", "AttributeType": "S"},
+            ],
+            KeySchema=[
+                {"AttributeName": "client", "KeyType": "HASH"},
+                {"AttributeName": "app", "KeyType": "RANGE"},
+            ],
+            BillingMode="PAY_PER_REQUEST",
+        )["TableDescription"]
+        dynamodb_client.update_continuous_backups(
+            TableName="test1",
+            PointInTimeRecoverySpecification={"PointInTimeRecoveryEnabled": True},
+        )
+        # DynamoDB client for this test class
+        audit_info = self.set_mocked_audit_info()
+        dynamo = DynamoDB(audit_info)
+        assert len(dynamo.tables) == 1
+        assert dynamo.tables[0].arn == table["TableArn"]
+        assert dynamo.tables[0].pitr
+
+    # Test DAX List Tables
+    @mock_dax
+    def test__describe_clusters__(self):
+        # Generate DAX Client
+        dax_client = client("dax", region_name=AWS_REGION)
+        # Create DAX Clusters
+        iam_role_arn = f"arn:aws:iam::{AWS_ACCOUNT_NUMBER}:role/aws-service-role/dax.amazonaws.com/AWSServiceRoleForDAX"
+        dax_client.create_cluster(
+            ClusterName="daxcluster1",
+            NodeType="dax.t3.small",
+            ReplicationFactor=3,
+            IamRoleArn=iam_role_arn,
+            SSESpecification={"Enabled": True},
+        )
+        dax_client.create_cluster(
+            ClusterName="daxcluster2",
+            NodeType="dax.t3.small",
+            ReplicationFactor=3,
+            IamRoleArn=iam_role_arn,
+            SSESpecification={"Enabled": True},
+        )
+        # DAX client for this test class
+        audit_info = self.set_mocked_audit_info()
+        dax = DAX(audit_info)
+        assert len(dax.clusters) == 2
```

Comment:
Assert the possible values for the `cliusters` object.

Reply:
Done